ui/src/services/utils.js

```diff
@@ -280,7 +280,13 @@ export function addMissingDataPoint(
   sampleDuration,
   sampleFrequency,
 ) {
-  if (!orginalValues || orginalValues.length === 0) {
+  if (
```

Comment:
could describe which edge cases are handled (in the commit message).

Reply:
hi @slaperche-scality, actually it's more props checking not edge cases, I reword my commit message